app/src/sharedTest/java/org/oppia/app/options/OptionsFragmentTest.kt

```diff
@@ -152,26 +175,44 @@ class OptionsFragmentTest {
       )
       onView(withId(R.id.story_text_size_seekBar)).perform(clickSeekBar(5))
       onView(withContentDescription(R.string.go_to_previous_page)).perform(click())
-      onView(atPositionOnView(R.id.options_recyclerview, 0, R.id.story_text_size_text_view)).check(
+      onView(
+        atPositionOnView(
+          R.id.options_recyclerview,
+          0, R.id.story_text_size_text_view
+        )
+      ).check(
         matches(withText("Medium"))
       )
     }
   }
 
+  /* ktlint-disable max-line-length */
   @Test
   fun testOptionFragment_clickStoryTextSize_changeTextSizeToMedium_changeConfiguration_checkTextSizeMediumIsSelected() {
     launch<OptionsActivity>(createOptionActivityIntent(0)).use {
-      onView(atPositionOnView(R.id.options_recyclerview, 0, R.id.story_text_size_item_layout)).perform(
+      onView(
+        atPositionOnView(
+          R.id.options_recyclerview,
+          0,
+          R.id.story_text_size_item_layout
+        )
+      ).perform(
         click()
       )
       onView(withId(R.id.story_text_size_seekBar)).perform(clickSeekBar(5))
       onView(isRoot()).perform(orientationLandscape())
       onView(withContentDescription(R.string.go_to_previous_page)).perform(click())
-      onView(atPositionOnView(R.id.options_recyclerview, 0, R.id.story_text_size_text_view)).check(
+      onView(
+        atPositionOnView(
+          R.id.options_recyclerview,
+          0, R.id.story_text_size_text_view
+        )
+      ).check(
         matches(withText("Medium"))
       )
     }
   }
+  /* ktlint-enable max-line-length */
```

Comment:
Ditto

Reply:
Done

Reply:
Resolving this on behalf of @vinitamurthi . Though it was not mentioned by her, but as the changes are are fairly simple and this PR has been approved below by @vinitamurthi.